cfi_csrs.adoc

```diff
@@ -32,7 +32,7 @@ With these set of controls the backward-edge and forward-edge CFI may be
 separately enforced at S-mode and for each application.
 ====
 
-=== Hypervisor environment configuration registers (`henvcfg and henvcfgh`)
+=== Hypervisor environment configuration registers (`menvcfg and henvcfg`)
```

Comment:
The registers in question are `henvcfg` and `henvcfgh` - please see section 8.2.5 of Priv. 
The update to call them `menvcfg` and `henvcfg` is not correct. 
Could you please update this PR to correct this.